src/components/SettingsModal.js

```diff
@@ -12,6 +20,36 @@ export default function SettingsModal({ setDisplayModal }) {
           className="modal-close-button"
         />
         <h1 className="modal-header">Settings</h1>
+
+        <div className="modal-range-container">
+          <label htmlFor="delay-slider">
+            Set how frequently the app will fetch new places. Values are in
+            seconds.
+          </label>
+          <div className="modal-range-pic-container">
+            <span className="modal-range-pic">
```

Comment:
Totally non-blocking, but you could consider putting your `<img/>` tags inside `<figure/>` tags for slightly better semantics.

Reply:
Totally. Whenever I use `div` and `span`, I always know there's a better option, but I often forget which replacements to utilize. Thanks for the reminder. 😺